sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SparkPlanGraph.scala

```diff
@@ -79,7 +79,7 @@ object SparkPlanGraph {
       subgraph: SparkPlanGraphCluster,
       exchanges: mutable.HashMap[SparkPlanInfo, SparkPlanGraphNode]): Unit = {
     planInfo.nodeName match {
-      case name if name.startsWith("WholeStageCodegen") =>
+      case name if name.startsWith("WholeStage") =>
```

Comment:
shall we make it more general? Maybe we can define a special metadata key to indicate clustering the plan nodes.